data/assets/scene/digitaluniverse/deepsky.asset

```diff
@@ -25,22 +25,16 @@ local DeepSkyObjects = {
       MinMaxSize = { 16, 20 },
       Unit = "pc"
     },
-    Color = { 1.0, 1.0, 0.0 },
     Opacity = 0.99,
-    ScaleFactor = 500.0,
     File = speck .. "dso.speck",
     Texture = textures .. "point3.png",
-    --ColorMap = speck .. "tully.cmap",
-    --ColorMap = speck .. "lss.cmap",
-    --ColorOption = { "proximity" },
-    --ColorOption = { "prox5Mpc" },
-    --ColorRange = { { 1.0, 30.0 } },
     Unit = "pc",
     --FadeInDistances = { 0.05, 1.0 }, -- Fade in value in the same unit as "Unit"
-    BillboardMinMaxSize = { 0.0, 8.22 }, -- in pixels
-    --CorrectionSizeEndDistance = 22.0,
-    --CorrectionSizeFactor = 10.45
-    EnablePixelSizeControl = true
+    SizeSettings = {
```

Comment:
OBS!  Something goes wrong with the sizing.  These look huge from Earth, and they didn't use to.  Should be fixed!

Reply:
Need to double-check this.